pkg/controller/internal/api_client.go

```diff
@@ -30,5 +30,5 @@ func BuildClient(ctx context.Context, client client.Client,
 		return nil, err
 	}
 
-	return h.NewRESTClientForHost(harbor.Spec.InstanceURL, "/api", "admin", corePassword), nil
+	return h.NewRESTClientForHost(harbor.Spec.InstanceHost+"/api", "admin", corePassword)
```

Comment:
The new client will parse this as an URL, so we can probably revert `InstanceHost` back to `InstanceURL` to support http and https endpoints?